src/components/templates/Auditorium/components/SectionPreviews/SectionPreviews.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useMemo } from "react";
+import { useHistory } from "react-router";
+import classNames from "classnames";
+
+import { AuditoriumSizes } from "types/auditorium";
+import { AuditoriumVenue } from "types/venues";
+
+import { useAuditoriumSections } from "hooks/auditoriumSections";
+import { useVenueId } from "hooks/useVenueId";
+
+import { SectionPreview } from "../SectionPreview";
+import { Video } from "../Video";
+
+import "./SectionPreviews.scss";
+
+export interface SectionPreviewsProps {
+  venue: AuditoriumVenue;
+}
+
+export const SectionPreviews: React.FC<SectionPreviewsProps> = ({ venue }) => {
```

Comment:
Is there a better name we can use for this to differentiate it from `SectionPreview` by more than `s`? I feel like this is a typo/misread waiting to happen.

Reply:
I agree, but I can't come up with a good one.

Reply:
If you were to describe what this component does in a sentence or two, what is it? That might help me/us come up with a better name for it.

Reply:
It's the view of all the section-previews. Basically, that is what user sees, when he enters the auditorium, before choosing a section

Reply:
And a 'section preview' is essentially the equivalent to the 'table view' that is displayed in the first screen of the JazzBar/etc?

---

How about just `AuditoriumSections` or similar? (or `AuditoriumSectionPreviews` if needed, though 'preview' feels like the wrong name to use on these components as well)

Or potentially `AllSections` / `AllSectionPreviews` or similar?

Can this ever be rendered outside of the `Auditorium` (and/or do we have future plans/intentions to?)

Reply:
changed here https://github.com/sparkletown/sparkle/pull/1251/commits/adf6f1b18663d0e43209d78822eb78a1f85af80d